Gopkg.toml

```diff
@@ -56,11 +60,11 @@ ignored = ["github.com/fnproject/fn/cli",
 
 [[constraint]]
   name = "github.com/fsouza/go-dockerclient"
-  revision = "1b33efcb40bef3494af8bc6a1f73600f37a3c9d1"
+  revision = "v1.3.0"
 
 [[constraint]]
   name = "github.com/coreos/go-semver"
-  version = "0.2.0"
+  revision = "e214231b295a8ea9479f11b70b35d5acf3556d9b"
```

Comment:
do we need the constraint on this dep? (we don't need constraints on every dep) - am not sure why this version but this dep is really minimal and just used for checking docker's version...

Reply:
I was told lock to this one.